analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ConfiguringLoggersTest.cs

```diff
@@ -95,16 +110,35 @@ public class ConfiguringLoggersTest
             NuGetMetadataReference.SerilogPackages(Constants.NuGetLatestVersion);
 
 #if NET
-        private static IEnumerable<MetadataReference> AspNetCoreLoggingReferences =>
+        private static IEnumerable<MetadataReference> AspNetCore2LoggingReferences =>
             NetStandardMetadataReference.Netstandard
```

Comment:
`NetStandardMetadataReference.Netstandard` is no longer needed. I will do a separate PR to remove it from most of the test cases.